Automattic/woocommerce-payments
# Update WordPress dev dependencies

Fixes #6085 
Fixes #6086
Fixes #6090  

#### Changes proposed in this Pull Request
- Update `@wordpress/babel-plugin-makepot` to v4.3.2
- Update `@wordpress/base-styles` to v4.3.1
- Update `@wordpress/jest-preset-default` to v8.1.2
- Update dependencies doc









#### Testing instructions





* Ensure JS tests are passing.
* Ensure language file is created after building release.
* Ensure referenced styles are present in `base-styles`.

-------------------


**Post merge**
